Week1/homework.js

```diff
@@ -0,0 +1,253 @@
+let mysql = require('mysql');
+
+// establish db
+let db = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  database: 'world',
```

Comment:
You might want to leave this line out. In case the database 'world' is not present in mysql, then this would give an error.